vpr/src/draw/draw.cpp

```diff
@@ -1772,8 +1772,8 @@ static void draw_rr_edges(int inode, ezgl::renderer* g) {
                         } else {
                             g->set_color(blk_DARKGREEN);
                         }
-                        switch_type = device_ctx.rr_nodes[inode].edge_switch(iedge);
-                        draw_chanx_to_chanx_edge(RRNodeId(inode), RRNodeId(to_node),
+                        switch_type = rr_graph.edge_switch(RRNodeId(inode), iedge);
```

Comment:
Can you use ``rr_node`` instead of ``RRNodeId(inode)`` here?

Reply:
OK sure.